src/Expecto.VisualStudio.TestAdapter/TestCases.fs

```diff
@@ -0,0 +1,18 @@
+﻿module TestCases
+
+open System.Linq
+open Microsoft.VisualStudio.TestPlatform.ObjectModel
+
+let fullyQualified (name:string) =
```

Comment:
`fullyQualified` and `traitName` could be marked as private since they are used only in `TestCases.fs`.